logging/structured/src/main/scala/tofu/logging/Logs.scala

```diff
@@ -73,3 +80,44 @@ object Logs {
       logs.forService[X].flatMap(f)
   }
 }
+
+private[logging] trait LogsInstances0 extends LogsInstances1 {
+  implicit def logs2ApplyK[Y[_]](implicit Y: Apply[Y]): ApplyK[Logs[Y, *[_]]] =
+    new Logs2ApplyK[Y] { def I: Apply[Y] = Y }
+}
+
+private[logging] trait LogsInstances1 {
+  implicit def logs2FunctorK[Y[_]](implicit Y: Functor[Y]): FunctorK[Logs[Y, *[_]]] =
+    new Logs2FunctorK[Y] { def I: Functor[Y] = Y }
+}
+
+trait Logs2FunctorK[Y[_]] extends FunctorK[Logs[Y, *[_]]] {
+  implicit def I: Functor[Y]
+
+  def mapK[F[_], G[_]](af: Logs[Y, F])(fk: F ~> G): Logs[Y, G] = new Logs[Y, G] {
+    def forService[Svc: ClassTag]: Y[Logging[G]] = af.forService[Svc].map(_.mapK(fk))
+    def byName(name: String): Y[Logging[G]]      = af.byName(name).map(_.mapK(fk))
+  }
+}
+
+trait Logs2ApplyK[Y[_]] extends Logs2FunctorK[Y] with ApplyK[Logs[Y, *[_]]] {
```

Comment:
Is this necessary to make them by hand?
tofu-logging depends on cats-tagless-macros, so one can `Derive` them

Reply:
Afaik, cats.tagless.Derive can't derive such instances. I tried but didn't manage to. Will double-check

Reply:
Sorry. My Bad.